### PR TITLE
adding the KMS samples for creating keys, keyrings, encrypt and decrypt

### PR DESCRIPTION
**Pull request by bshaffer**
todo: 
 - [x] add tests
 - [ ] write samples for other KMS functions
 - [x] add README

### CODE REVIEW

File: kms/api/decrypt.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+$ciphertext = file_get_contents($infile);
+
+$request = new Google_Service_CloudKMS_DecryptRequest();
+$request->setCiphertext($ciphertext);
+$response = $kms->projects_locations_keyRings_cryptoKeys->decrypt(
+    $keyName,
+    $request
+);
+
+// The plaintext response comes back base64 decoded.
```
Comment (tmatsuo) [inline kms/api/decrypt.php]: I think the comment here should be:

```
// The plaintext response comes back base64 encoded.
```

**Comment by bshaffer** [comment]
Done! PTAL

File: kms/api/test/KmsTest.php [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+        $client->setScopes([
+            'https://www.googleapis.com/auth/cloud-platform'
+        ]);
+        $this->kms = new Google_Service_CloudKMS($client);
+    }
+
+    public function testCreateCryptoKey()
+    {
+        if (!$keyRing = getenv('GOOGLE_KMS_KEYRING')) {
+            return $this->markTestSkipped('Set the GOOGLE_KMS_KEYRING environment variable');
```
Comment (tmatsuo) [inline kms/api/test/KmsTest.php]: Can you run all the tests on travis? (or in other words, have you set the environment variables)

Comment (bshaffer) [inline kms/api/test/KmsTest.php]: All set up!

**Pull request merged**